server/routes/search.ts

```diff
@@ -19,51 +19,44 @@ export default function routes({ auditService, prisonerSearchService, visitServi
       handlers.map(handler => asyncMiddleware(handler)),
     )
 
-  get(['/prisoner', '/prisoner-visit'], (req, res) => {
+  get(['/prisoner'], (req, res) => {
     const search = req?.body?.search
 
-    const isVisit = req.originalUrl.includes('-visit')
-    let establishmentHref = '/search/prisoner/'
-    if (isVisit) {
-      establishmentHref = '/search/prisoner-visit/'
-    }
-
     res.render('pages/search/prisoner', {
       search,
       visit: req.originalUrl.includes('-visit'),
       showEstablishmentSwitcher: true,
-      establishmentHref,
     })
   })
 
-  post(['/prisoner', '/prisoner-visit'], body('search').trim('. '), (req, res) => {
-    const isVisit = req.originalUrl.includes('-visit')
+  post(['/prisoner'], body('search').trim('. '), (req, res) => {
     const { search } = req.body
 
     return res.redirect(
       url.format({
-        pathname: `/search/prisoner${isVisit ? '-visit' : ''}/results`,
+        pathname: `/search/prisoner/results`,
         query: {
           ...(search && { search }),
         },
       }),
     )
   })
 
-  get(['/prisoner/results', '/prisoner-visit/results'], async (req, res) => {
+  get(['/prisoner/results'], async (req, res) => {
     const { prisonId } = req.session.selectedEstablishment
-    const isVisit = req.originalUrl.includes('-visit')
     const search = typeof req.query.search === 'string' ? req.query.search : ''
+
     const currentPage = typeof req.query.page === 'string' ? req.query.page : ''
     const parsedPage = Number.parseInt(currentPage, 10) || 1
     const { pageSize } = config.apis.prisonerSearch
+
     const validationErrors = validatePrisonerSearch(search)
     const errors = validationErrors ? [validationErrors] : []
     const hasValidationErrors = errors.length > 0
 
     const { results, numberOfPages, numberOfResults, next, previous } = hasValidationErrors
       ? { results: 0, numberOfPages: 0, numberOfResults: 0, next: 0, previous: 0 }
-      : await prisonerSearchService.getPrisoners(search, prisonId, res.locals.user.username, parsedPage, isVisit)
+      : await prisonerSearchService.getPrisoners(search, prisonId, res.locals.user.username, parsedPage, false) // todo
```

Comment:
Better commenting method for TODOs is use a full line:

`// TODO: Blah blah`

Reply:
And what it is you're TODOing as there's no context in that comment, I don't know what you're todoing here.

Reply:
I did my usual todo search (it was meant for me to come back to) but had context set from previous searches so missed it! Will make it clearer in the future (and also hopefully find it myself)

Reply:
Coincidentally, I recently came across this VSCode extension I've been trying out (https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree) that can highlight things like `TODO`s. Quite handy for keeping track of comments like that while you're in the middle of a chunkier piece of work.